### ISSUE TITLE
result of `gulp site` seems broken

### ISSUE THREAD

**Issue by shrugs** [urbit/indigo#26]
<img width="1654" alt="image" src="https://user-images.githubusercontent.com/1535001/59378420-1b139100-8d09-11e9-97de-f7113a828332.png">

```bash
git clone ...
cd indigo
npm install
gulp update-site-indigo
gulp site
```

the layout seems to be relatively broken in the demo site — anything I'm missing?

**Comment by jyng** [comment]
The build of the site is currently branch `improvements`. I can merge that in if need be.

**Comment by shrugs** [comment]
@jyng up to y'all, but would be good for me to be able to work off master without having to worry about important changes in random branches

**Comment by jyng** [comment]
Cool I'll do this right now

**Comment by shrugs** [comment]
closed by #27
